src/nspawn/nspawn.c

```diff
@@ -286,6 +287,7 @@ static void help(void) {
                "     --link-journal=MODE    Link up guest journal, one of no, auto, guest, \n"
                "                            host, try-guest, try-host\n"
                "  -j                        Equivalent to --link-journal=try-guest\n"
+               "     --resolv-conf=MODE     Select mode of /etc/resolv.conf synchronization\n"
```

Comment:
synchronization → initialization to underline the fact that there is no synchronization actually?